.circleci/config.yml

```
@@ -78,6 +83,22 @@ jobs:
             dbt --warn-error run --target snowflake --full-refresh
             dbt --warn-error run --target snowflake
 
+      - run:
+         name: "Run Tests - BigQuery"
```

hope this is the final piece
```suggestion
         name: "Run Tests - BigQuery"
         environment:
              BIGQUERY_SERVICE_KEY_PATH: "/home/circleci/bigquery-service-key.json"
```